PurchasesTests/Attribution/AttributionTypeFactoryTests.swift

```diff
@@ -42,7 +42,37 @@ class AttributionTypeFactoryTests: XCTestCase {
 
     func testCanRotateAdvertisingIdentifierBack() {
         let expected = "advertisingIdentifier"
-        let randomized = "nqiregvfvatVqragvsvre"
```

Comment:
the tests hardcoded the values here, so if you changed the values in the main implementation the tests would still pass, which felt worrisome. 
I made the values available to the tests instead.

Reply:
yes, good point